streamprocessor/src/main/java/org/md2k/streamprocessor/output/puffMarkerFeatureVector.java

```diff
@@ -0,0 +1,382 @@
+package org.md2k.streamprocessor.output;
+
+import android.content.Context;
+
+import org.md2k.datakitapi.datatype.DataTypeDoubleArray;
+import org.md2k.datakitapi.source.METADATA;
+import org.md2k.datakitapi.source.datasource.DataSourceBuilder;
+import org.md2k.datakitapi.source.datasource.DataSourceType;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/*
+ * Copyright (c) 2016, The University of Memphis, MD2K Center
+ * - Timothy Hnat <twhnat@memphis.edu>
+ * All rights reserved.
+ * 
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * 
+ * * Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ * 
+ * * Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ * 
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+public class puffMarkerFeatureVector extends Output {
+    public puffMarkerFeatureVector(Context context) {
+        super(context, DataSourceType.PUFFMARKER_FEATURE_VECTOR);
+    }
+
+    ArrayList<HashMap<String, String>> createDataDescriptor() {
+        ArrayList<HashMap<String, String>> result = new ArrayList<HashMap<String, String>>();
+
+        HashMap<String, String> dataDescriptor;
+
+        // (1)
+        dataDescriptor = new HashMap<>();
```

Comment:
Use cStressFeatureVector.java as a reference.  These entries should have METADATA.DESCRIPTIVESTATISTICS set as appropriate and their NAME changed as appropriate.